pkg/skaffold/color/formatter.go

```diff
@@ -103,12 +102,8 @@ func isTerminal(w io.Writer) bool {
 		return true
```

Comment:
Is the function comment still current?  Or could we just replace `isTerminal` with a direct call to `IsTerminal()`?

Reply:
I'm removing the comment. The test on `ColoredWriteCloser` is needed here